test/agent/microshift-test-agent.sh

```diff
@@ -108,8 +108,9 @@ EOF
 # When going from "just RHEL" to RHEL+MicroShift+dependencies there might be a
 # problem with groups. Following line ensures that sshd's keys are owned by
 # ssh_keys group and not some other.
+# It should not fail if no file matching wildcard exist.
 # When removing, change Before= in microshift-test-agent.service
-chown -v root:ssh_keys /etc/ssh/ssh_host*key
+chown -v root:ssh_keys /etc/ssh/ssh_host*key || true
```

Comment:
Let's try to delete this w/a. It may not be needed anymore because we're correctly using the same parent for subsequent ostree updates.